run.go

```diff
@@ -22,12 +22,14 @@ type RunOpts struct {
 	Env        map[string]string
 	Size       string
 	Detached   bool
+	Async      bool
 	HasUploads bool
 }
 
 type RunRes struct {
-	Container *Container `json:"container"`
-	AttachURL string     `json:"attach_url"`
+	Container    *Container `json:"container"`
+	AttachURL    string     `json:"attach_url"`
+	OperationURL string     `json:"operation_url"`
```

Comment:
There is no need for a JSON tag here I think. We don't JSON decode the operation URL.

```suggestion
	OperationURL string     `json:"-"`
```